code/modules/mapping/map_template.dm

```diff
@@ -79,8 +79,8 @@
 		affected_turf.levelupdate()
 
 /datum/map_template/proc/load_new_z()
-	var/x = round((world.maxx - width)/2)
-	var/y = round((world.maxy - height)/2)
+	var/x = round((world.maxx - width)/2) + 1
+	var/y = round((world.maxy - height)/2) + 1
```

Comment:
```suggestion
	var/x = round((world.maxx - width) * 0.5) + 1
	var/y = round((world.maxy - height) * 0.5) + 1
```

Reply:
Let's just clean this one up while we're here.

Reply:
?

Reply:
huh?

Reply:
I was under the impression that we preferred multiplying rather than dividing due to speed. I've been explicitly asked on old PRs to do multiplication instead of division where possible and double checked on the main discord before I did the change request.

![image](https://user-images.githubusercontent.com/24975989/94992130-bd4f2c00-057f-11eb-9767-6c12cb0543c7.png)